tycho/tycho.kicad_pro

```diff
@@ -648,7 +648,7 @@
     ]
   ],
   "text_variables": {
-    "DATE": "2024-03-07",
-    "VERSION": "r2.0.1"
+    "DATE": "2024-04-09",
+    "VERSION": "r2.0.1+"
```

Comment:
This is fine, but in the future please try to remember to update the version with a "+" before making any other changes.